domestic/templates/domestic/base.html

```diff
@@ -23,7 +26,13 @@
           <meta id="privacyCookieDomain" value=".great.gov.uk" />
         {% endblock %}
         <title>
-            {% block head_title %}{% endblock %}
+            {% block head_title %}
+                {% if page.seo_title %}
+                    {{page.seo_title}}
+                {% else %}
+                    {{page.title}}
```

Comment:
I can see that other pages - eg article_page.html have this as their `head_title` content:

{% block head_title %}{{ page.<name of title field here> }} - great.gov.uk{% endblock %}

so I think it's worth adding that - `- great.gov.uk` suffix to things:
```suggestion
                    {{page.title}} - great.gov.uk
```

Reply:
Done